build.py

```diff
@@ -305,7 +305,7 @@ def build_package(target, build_native_package):
     fmt_libcxx_rpath = os.path.join(fmt_libcxx_install_prefix, 'lib')
 
     avro_info = get_versions()['avro']
-    avro_subdirectory = '{0}{1}-{2}'.format('avro', avro_info['version_string'], avro_info['consortium_build_number'])
+    avro_subdirectory = '{0}{1}-{2}'.format('avro-libcxx', avro_info['version_string'], avro_info['consortium_build_number'])
```

Comment:
Should this change be happening for `avro_libcxx_subdirectory` instead?

Also noticed that avro-libcxx is not getting a package_revision bump even though it is changing. Why?

Reply:
> Should this change be happening for `avro_libcxx_subdirectory` instead?

Yes, thank you.

> Also noticed that avro-libcxx is not getting a package_revision bump even though it is changing. Why?

The only packages getting rebuilt are the ones that use the substitutions created by these variables.
Substitions:
- `TEMPLATE_QPID_PROTON_LIBCXX_RPATH`
    - Used by `qpid-proton-libcxx`
- `TEMPLATE_AVRO_LIBCXX_RPATH`
    - Not used by any package. We can probably get rid of it. It's only kept around on the off chance we want to migrate `mungefs` to libc++ or create a `-libcxx`-variant of `mungefs`
- `TEMPLATE_AVRO_LIBCXX_PATH`
    - Not used by any package. We can probably get rid of it. It's only kept around on the off chance we want to migrate `mungefs` to libc++ or create a `-libcxx`-variant of `mungefs`
- `TEMPLATE_ZMQ_LIBCXX_RPATH`
    - Used by `zeromq4-1-libcxx`

Since the `TEMPLATE_AVRO_CXX_` substitutions are never used, there's no package to revision bump.

Reply:
Ohh, I see. Thanks for the explanation.